OData/src/System.Web.OData/OData/Builder/EdmModelHelperMethods.cs

```diff
@@ -548,7 +564,15 @@ private static void AddCoreVocabularyAnnotations(this EdmModel model, Navigation
 
             if (edmProperties.Any())
             {
-                model.SetOptimisticConcurrencyAnnotation(target, edmProperties);
+                // todo: fix SetOptimisticConcurrencyAnnotation to support setting concurrency annotations on singletons
```

Comment:
Created [https://github.com/OData/odata.net/issues/770](https://github.com/OData/odata.net/issues/770).

---
In reply to: [105059250](https://github.com/OData/WebApi/pull/931#discussion_r105059250) [](ancestors = 105059250)